File: frontend/src/src/app/section/admin/admin-administration/dictionary.ts
export class Dictionary {

  private _id: number;
  private _addDate: Date;
  private _editDate: Date;
  private _key: string;
  private _value: string;
  private _image: string;

  constructor() {
  }

  get id(): number {
    return this._id;
  }

  get addDate(): Date {
    return this._addDate;
  }

  get editDate(): Date {
    return this._editDate;
  }

  get key(): string {
    return this._key;
  }

  get value(): string {
    return this._value;
  }

  get image(): string {
    return this._image;
  }

}

File: backend/src/main/java/com/com/backend/model/Dictionary.java
package com.com.backend.model;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import org.hibernate.annotations.Type;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Lob;
import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@ToString
@Entity
public class Dictionary extends AbstractEntity {

    @Column(nullable = false, columnDefinition = "timestamp default CURRENT_TIMESTAMP", updatable = false)
    private LocalDateTime addDate;
    @Column(nullable = false, columnDefinition = "timestamp default CURRENT_TIMESTAMP")
    private LocalDateTime editDate;
    @Column(nullable = false, updatable = false)
    private String key;
    private String value;
    @Lob
    @Type(type="org.hibernate.type.BinaryType")
    private byte[] image;

}
